SMLHelper/Handlers/CoordinatedSpawnsHandler.cs

```diff
@@ -228,11 +279,11 @@ public override int GetHashCode()
             unchecked // overflow is fine, just wrap around
             {
                 int hash = 13;
-                hash = (hash * 7) + TechType.GetHashCode();
-                hash = (hash * 7) + ClassId.GetHashCode();
-                hash = (hash * 7) + SpawnPosition.GetHashCode();
-                hash = (hash * 7) + Rotation.GetHashCode();
-                hash = (hash * 7) + Type.GetHashCode();
+                hash += (hash * 7) + TechType.GetHashCode();
+                hash += (hash * 7) + ClassId.GetHashCode();
+                hash += (hash * 7) + SpawnPosition.GetHashCode();
+                hash += (hash * 7) + Rotation.GetHashCode();
+                hash += (hash * 7) + Type.GetHashCode();
```

Comment:
What's the reason for this? Was it broken? Every hash code algorithm I can find from a quick Googling doesn't use `+=` here.

```suggestion
                hash = (hash * 7) + TechType.GetHashCode();
                hash = (hash * 7) + ClassId.GetHashCode();
                hash = (hash * 7) + SpawnPosition.GetHashCode();
                hash = (hash * 7) + Rotation.GetHashCode();
                hash = (hash * 7) + Type.GetHashCode();
```

Reply:
This was a test. We weren't sure if it was broken or not.
FYI this wasn't the issue, but I forgot to revert this change.